content/applications/finance/accounting.rst

```diff
@@ -15,17 +12,188 @@ development with features such as AI-powered invoice recognition, synchronizatio
 accounts, smart matching suggestions, etc.
 
 .. seealso::
-   - `Odoo Tutorials: Accounting <https://www.odoo.com/slides/accounting-19>`_
-   - :doc:`Accounting Cheat Sheet <accounting/getting_started/cheat_sheet>`
+   `Odoo Tutorials: Accounting <https://www.odoo.com/slides/accounting-19>`_
 
+.. cards::
+
+   .. card:: Getting started
+      :target: accounting/getting_started
+      :large:
+
+      Basic concepts of accounting and initial setup of your accounting
+
+   .. card:: Accounts receivable
+      :target: accounting/receivables
+
+      Customer invoices, payment terms, and electronic invoicing
+
+   .. card:: Accounts payable
+      :target: accounting/payables
+
+      Vendor bills, assets, and invoice digitization (OCR)
+
+   .. card:: Payments
+      :target: accounting/payments
+
+      Invoices and bills payments (online, checks, batches) and follow-up on invoices
+
+   .. card:: Bank and cash accounts
+      :target: accounting/bank
+
+      Bank synchronization, reconciliation, and cash registers
+
+   .. card:: Taxes
+      :target: accounting/taxes
+
+      Taxes, fiscal positions, and integrations
+
+   .. card:: Reporting
+      :target: accounting/reporting
+
+      Reporting, declarations, and analytic accounting
+
+Double-entry bookkeeping
+========================
+
+Odoo automatically creates all the behind-the-scenes journal entries
+for each of your accounting transactions: customer invoices, point of
+sale order, expenses, inventory moves, etc.
+
+Odoo uses the rules of double-entry bookkeeping system: all journal
+entries are automatically balanced (sum of debits = sum of credits).
+
+.. seealso::
+   - :doc:`Understand Odoo's accounting transactions per document
+     <accounting/getting_started/cheat_sheet>`
+
+Accrual and Cash Basis Methods
+==============================
+
+Odoo supports both accrual and cash basis reporting. This allows you to
```

Comment:
considering that this seems like a repetitive error - please adhere the comment re: the removal of "you/your" to this entire PR. thanks